core-bundle/src/Security/DataContainer/DataContainerSubject.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Security\DataContainer;
+
+class DataContainerSubject
+{
+    public function __construct(public readonly string $table, public readonly int|string|null $id = null, public readonly array $attributes = [])
+    {
+    }
+
+    public function __toString(): string
```

Comment:
from the PHP docs:

> As of PHP 8.0.0, any class that contains a [__toString()](dfile:///Users/aschempp/Library/Application%20Support/Dash/DocSets/PHP/PHP.docset/Contents/Resources/Documents/www.php.net/manual/en/language.oop5.magic.html#object.tostring) method will also implicitly implement the [Stringable](dfile:///Users/aschempp/Library/Application%20Support/Dash/DocSets/PHP/PHP.docset/Contents/Resources/Documents/www.php.net/manual/en/class.stringable.html) interface, and will thus pass type checks for that interface. Explicitly implementing the interface anyway is recommended.

Reply:
I know, but that's CS to me. We should do it everywhere in one specific commit.